crowbar_framework/app/controllers/api/crowbar_controller.rb

```diff
@@ -41,6 +41,7 @@ def upgrade
     end
   rescue Crowbar::Error::StartStepRunningError,
          Crowbar::Error::StartStepOrderError => e
+    log_exception(e)
```

Comment:
Are you sure it belongs here? I think we are able to handle thise Upgrade related exceptions pretty well, and the problematic part is really with `StandardError` only